pkg/metrics/collector/helper.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"io/ioutil"
+	"path"
+	"strconv"
+	"strings"
+)
+
+func readUintFromFile(path string) (uint64, error) {
```

Comment:
docstring

Reply:
Updated.

Reply:
you need to follow the go style comment. Make sure your code passes golint and govet

// readUintFromFile reads a uint ...